### PR TITLE
Jetpack: update Backup success image

### PR DESCRIPTION
**Pull request by keoshi**
#### Changes proposed in this Pull Request

* Update the Backup success image to the standard Material icon `cloud_done_24px` instead of using a custom icon.

#### Testing instructions

* Fire up this PR.
* Go to Calypso Green: http://jetpack.cloud.localhost:3000/
* Select a site with a Backup plan, and previous successful backups.
* Ensure you see the new icon.

#### Screenshots

Before | After
--|--
image | image

### CODE REVIEW

**Review by monsieur-z — APPROVED** [review]
LGTM! You might need to rebase with `trunk` to make the checks pass. I had similar issues recently.

File: client/components/jetpack/daily-backup-status/status-card/icons/cloud-success.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg width="50" height="52" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M39.948 21.38C38.544 14.033 32.29 8.518 24.774 8.518c-5.966 0-11.148 3.492-13.729 8.603C4.831 17.802 0 23.232 0 29.812 0 36.861 5.554 42.59 12.387 42.59h26.839c5.698 0 10.322-4.77 10.322-10.647 0-5.622-4.232-10.18-9.6-10.562z" fill="#069E08"/><path fill-rule="evenodd" clip-rule="evenodd" d="M22.67 30.393l-5.86-4.905-1.586 2.015 7.813 6.54 9.51-12.089-1.953-1.634-7.925 10.073z" fill="#fff"/></svg>
\ No newline at end of file
+<svg width="50" height="50" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M40.313 20.583C38.895 13.396 32.582 8 25 8c-6.02 0-11.25 3.417-13.854 8.417C4.875 17.083 0 22.396 0 28.833c0 6.896 5.604 12.5 12.5 12.5h27.083c5.75 0 10.417-4.666 10.417-10.416 0-5.5-4.27-9.959-9.688-10.334z" fill="#069E08"/><path d="M20.833 35.083l-7.291-7.291 2.937-2.938 4.354 4.354 10.792-10.791 2.938 2.937-13.73 13.73z" fill="#fff"/></svg>
```
Comment (jeffgolenski) [inline client/components/jetpack/daily-backup-status/status-card/icons/cloud-success.svg:1]: Do we want `fill="#fff"` inline within the svg code or should we be keeping all style values in the css? Either way the change looks good to me!

**Review by jeffgolenski — CHANGES_REQUESTED** [review]

File: client/components/jetpack/daily-backup-status/status-card/icons/cloud-success.svg (hunk 1 above)
Comment (keoshi) [reply client/components/jetpack/daily-backup-status/status-card/icons/cloud-success.svg:1]: That hasn't changed with this PR and we're already using `fill="#069E08"` for the cloud itself. Plus, we can always override these via CSS if they end up being used in a new scenario where the default values don't work in our favor.

**Review by jeffgolenski — APPROVED** [review]
I hereby deem thee, on this last Friday of March in the year 2021, APPROVED for mergification into the great Jetpack Abyss. Carry forth with your head held high @keoshi. You have done a great service.

**Pull request merged**